solutions/silver/usaco-1254.mdx

```diff
@@ -27,7 +27,7 @@ using namespace std;
 
 #define f first
 #define s second
-constexpr int MAX_N = 200'001;
+const int MAX_N = 2e5;
```

Comment:
you sure the code still works?

Reply:
yep, submitted to usaco to be sure, it works.